en/mozillavpn.xlf

```diff
@@ -0,0 +1,505 @@
+    </trans-unit>
+    <trans-unit id="activeVsMaxDeviceCount">
+      <source xml:space="preserve">%1 of %2</source>
+      <context-group purpose="location"><context context-type="linenumber">45</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">53</context></context-group>
+      <note annotates="source" from="developer">Context: You have "x of y" (y is max) devices in your account.</note>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/components/VPNControllerServer.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="selectLocation">
+      <source xml:space="preserve">Select location</source>
+      <context-group purpose="location"><context context-type="linenumber">25</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/views/ViewServers.qml</context><context context-type="linenumber">17</context></context-group>
+      <note annotates="source" from="developer">Select the Location of the VPN server</note>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/components/VPNControllerView.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.deactivated">
+      <source xml:space="preserve">VPN is off</source>
+      <context-group purpose="location"><context context-type="linenumber">42</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">80</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">284</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.activationSloagan">
+      <source xml:space="preserve">Turn on to protect your privacy</source>
+      <context-group purpose="location"><context context-type="linenumber">48</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">85</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">290</context></context-group>
+    </trans-unit>
+    <trans-unit id="connecting">
+      <source xml:space="preserve">Connecting&#8230;</source>
+      <context-group purpose="location"><context context-type="linenumber">119</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.activating">
+      <source xml:space="preserve">Masking connection and location</source>
+      <context-group purpose="location"><context context-type="linenumber">125</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.activated">
+      <source xml:space="preserve">VPN is on</source>
+      <context-group purpose="location"><context context-type="linenumber">163</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.active">
+      <source xml:space="preserve">Secure and private  &#8226;  </source>
```

Comment:
Besides the note about using the utf8 character directly, why is there a bullet at the end of this string? Why so many spaces around it?